src/index.js

```diff
@@ -0,0 +1,117 @@
+import './style.css';
+import Checkbox from './checkbox.js';
+import Storage from './inputRecieve.js';
+import Tasks from './task.js';
+
+const tasksDisplay = document.querySelector('.tasks');
+
+class Ui {
+  static displayTasks() {
+    const tasks = Storage.getStorage();
+
+    tasks.forEach((task) => {
+      this.render(task);
+    });
+  }
+
+  static render(task) {
+    tasksDisplay.innerHTML += `
+    <li class="displayList">
+      <div class="innerItems">
+        <input class="checkbox" type="checkbox">
+        <p class="inputDisplay" contentEditable = true>${task.description}</p>
+      </div>
+      <div class="innerItems">
+        <button class="listButton"><i class='fas fa-ellipsis-v'></i></button>
+        <button class="binButton"><i class="material-icons" id="bin">delete</i></button>
+      </div>
+    </li>
```

Comment:
[optional] please use dynamic Java script like creat element it easy to maintain and easy ready